frontend/public/components/app.jsx

```diff
@@ -358,6 +380,19 @@ const PollConsoleUpdates = React.memo(function PollConsoleUpdates() {
     setIsFetchingPluginEndpoints(true);
   }
 
+  const pluginManifestsVersionsChanged = pluginManifestsData?.reduce((acc, obj) => {
+    prevPluginManifestsData?.forEach((o) => {
+      if (obj.name === o.name && obj.version !== o.version) {
+        acc.push(obj);
+      }
+    });
+    return acc;
+  }, []);
+  const pluginManifestsChanged = !_.isEmpty(pluginManifestsVersionsChanged);
```

Comment:
```suggestion
  const pluginManifestsVersionsChanged = pluginManifestsData?.some((manifest) => {
    return prevPluginManifestsData?.some((previousManifest) => {
      return manifest.name === previousManifest.name && manifest.version !== previousManifest.version;
    });
  });
```

Reply:
This is a little inefficient with the nested loops, but since the plugin list should be small, I think it's OK.